modules/cluster-management/src/main/java/org/apache/ignite/internal/cluster/management/ClusterTag.java

```diff
@@ -45,7 +45,22 @@ public interface ClusterTag extends NetworkMessage, Serializable {
      * @param name Cluster name.
      * @return Cluster tag instance.
      */
-    static ClusterTag clusterTag(CmgMessagesFactory msgFactory, String name) {
+    static ClusterTag randomClusterTag(CmgMessagesFactory msgFactory, String name) {
+        return msgFactory.clusterTag()
+                .clusterName(name)
+                .clusterId(UUID.randomUUID())
+                .build();
+    }
+
+    /**
+     * Creates a new cluster tag instance. Acts like a constructor replacement.
+     *
+     * @param msgFactory Message factory to instantiate builder.
+     * @param name Cluster name.
+     * @param clusterId Cluster ID.
+     * @return Cluster tag instance.
+     */
+    static ClusterTag clusterTag(CmgMessagesFactory msgFactory, String name, UUID clusterId) {
         return msgFactory.clusterTag()
                 .clusterName(name)
                 .clusterId(UUID.randomUUID())
```

Comment:
clusterID is ignored. Strange, that IDEA doesn't grey out unused parameters

Reply:
Good catch!